mod/wikirate/set/all/export.rb

```diff
@@ -1,50 +0,0 @@
-format :json do
```

Comment:
most of this was already a duplicate of decko code.  new refactoring obviates it further.